_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -115,7 +203,7 @@ This `div` does not have a [semantic role](#semantic-role).
 
 #### Inapplicable Example 2
 
-This `checkbox` has an [implicit semantic role](#implicit-role) that is identical to the [explicit semantic role](#explicit-role).
+This `checkbox` has an [implicit semantic role](#implicit-role) that is identical to the [explicit semantic role](#explicit-role). This allows native HTML `checked` state to apply.
```

Comment:
Technically, that should probably be the "`checked` content attribute" (state/property are an ARIA notion). Or maybe even the [checkedness](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#concept-fe-checked) (which is the same as the `checked` IDL attribute).

Technicalities aside, I'd still prefer
```suggestion
This `checkbox` has an [implicit semantic role](#implicit-role) that is identical to the [explicit semantic role](#explicit-role). This allows native HTML `checked` attribute to apply.
```